channel_test.go

```diff
@@ -102,23 +102,22 @@ func TestMessageOversize(t *testing.T) {
 	}()
 
 	_, _, err := rch.recv()
-	if err == nil {
-		t.Fatalf("error expected reading with small buffer")
-	}
-
-	status, ok := status.FromError(err)
-	if !ok {
-		t.Fatalf("expected grpc status error: %v", err)
-	}
-
-	if status.Code() != codes.ResourceExhausted {
-		t.Fatalf("expected grpc status code: %v != %v", status.Code(), codes.ResourceExhausted)
+	if err != nil {
```

Comment:
This test shouldn't branch, if an error is expected this should assert the error is not nil.